db/migrate/20221219205356_archive_needs_help2017_column.rb

```diff
@@ -0,0 +1,18 @@
+class ArchiveNeedsHelp2017Column < ActiveRecord::Migration[7.0]
+  def change
+    add_column :intake_archives, :needs_help_2017, :integer
+    reversible do |direction|
+      direction.up do
+        safety_assured do
+          # Safe for migrations because this is an INSERT into a column we are creating in this transactional migration.
+          execute <<~SQL
+            INSERT INTO intake_archives(id, needs_help_2017)
+              (SELECT id, needs_help_2017 from intakes)
```

Comment:
Is this fast? seems important to know as part of assessing this approach

Reply:
No - it takes well over a minute on my database replica I created with 30 GB of RAM (like the real one) and it hasn't even completed yet. :(

Reply:
```
db=# INSERT INTO intake_archives(id, needs_help_2017)
        (SELECT id, needs_help_2017 from intakes)
    ON CONFLICT (id) DO
UPDATE SET needs_help_2017=EXCLUDED.needs_help_2017;
INSERT 0 2629459
Time: 173579.771 ms (02:53.580)
```

I guess it depends if you think 3 minutes is fast or not.

Reply:
```
db=# INSERT INTO intake_archives(id, needs_help_2017)
        (SELECT id, needs_help_2017 from intakes)
    ON CONFLICT (id) DO
UPDATE SET needs_help_2017=EXCLUDED.needs_help_2017;
INSERT 0 2629459
Time: 58860.453 ms (00:58.860)
```

Faster the second time :)